libsoftwarecontainer/component-test/softwarecontainerlib_componenttest.cpp

```diff
@@ -44,18 +45,20 @@ class SoftwareContainerApp : public SoftwareContainerLibTest
 
 public:
 
-    void startGateways(const std::string &config, const std::string &gatewayID)
+    bool startGateways(const std::string &config, const std::string &gatewayID)
     {
         json_error_t error;
         json_t *configJson = json_loads(config.c_str(), 0, &error);
-        ASSERT_FALSE(configJson == nullptr);
+        assert(configJson != nullptr);
```

Comment:
This change is because gtest doesn't seem to handle ASSERT_* macros outside of a TEST_* macro very well. And also, I don't think we want to test that the config is valid etc., since that's not the point of the tests (the tests contain the config themselves), we just want to assert that the test has the required prerequisites to work correctly.